all_repos/push/github_pull_request.py

```diff
@@ -12,13 +14,18 @@ class Settings(NamedTuple):
     username: str
     fork: bool = False
     base_url: str = 'https://api.github.com'
+    git_https_re: Pattern[str] = re.compile(r'http?s:\/\/.*?\/(.*)')
```

Comment:
I don't think you need to escape forward slashes in regexes (only in javascript) -- also I think this is supposed to be `https?` ? (right now it matches `htts://`)

maybe also `[^/]+/(.*)$` would be slightly better for the matching portion too